yellowbrick/utils/timer.py

```diff
@@ -0,0 +1,49 @@
+# yellowbrick.utils.timer
+# Timer utilities
+#
+# Author:   ZJ Poh <poh.zijie@gmail.com>
+# Created:  Mon Jul 16 10:51:13 2017 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+"""
+Timer utilities
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import time
+
+##########################################################################
+## Timer Class
+##########################################################################
+
+
+def human_readable_time(s):
+    h, s = divmod(s, 3600)
+    m, s = divmod(s, 60)
+    return "{:>02.0f}:{:02.0f}:{:>07.4f}".format(h, m, s)
```

Comment:
This is a great function that doesn't require the python-dateutil library; thanks!